js/tabs.js

```diff
@@ -465,26 +469,13 @@ Tabs.prototype.saveEntry_ = function(tab, entry, opt_callback) {
   this.save(tab, opt_callback);
 };
 
-Tabs.prototype.onDocChanged_ = function(e, session) {
+Tabs.prototype.onDocChanged_ = function(_, { type }) {
```

Comment:
the new arguments here are hard to read. can we keep e and pass a named object instead? adding jsdoc would help

Reply:
sure